src/reason_pprint_ast.ml

```diff
@@ -697,6 +699,16 @@ let rules = [
   [
     (TokenPrecedence, (fun s -> (Nonassoc, s = "=>")));
   ];
+  [
+    (TokenPrecedence, (fun s ->
+      let is_infix_op = Hashtbl.mem infix_operators s in
```

Comment:
Just curious why do you have this mechanism to check if a infix_operator is registered or not. Why not just do
(TokenPrecedence, (fun s -> (Left, s.[0] = "~"))); ?